File: beginner/1113.c
# include <stdio.h>

int main(){
	int n1 = 0, n2 = 0;
	
	while(1){
		scanf("%i", &n1);
		scanf("%i", &n2);
		
		if( n1 == n2 )
			break;
		
		if( n1 > n2 )
			printf("Decrescente\n");
		
		if( n2 > n1 )
			printf("Crescente\n");
	}
	
	return 0;
}

File: beginner/1113.py
while True:
	n1, n2 = input().split(" ")
	n1, n2 = int(n1), int(n2)
	if n1 == n2:
		break
	if n1 > n2:
		print('Decrescente')
	if n2 > n1:
		print('Crescente')
